lib/maplibre-compose-material3/src/commonMain/kotlin/dev/sargunv/maplibrecompose/material3/Dp.kt

```diff
@@ -0,0 +1,16 @@
+package dev.sargunv.maplibrecompose.material3
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.unit.DpOffset
+
+@Composable
+@ReadOnlyComposable
+internal fun Offset.toDpOffset(): DpOffset =
+  with(LocalDensity.current) { DpOffset(x.toDp(), y.toDp()) }
+
+@Composable
+@ReadOnlyComposable
+internal fun DpOffset.toOffset(): Offset = with(LocalDensity.current) { Offset(x.toPx(), y.toPx()) }
```

Comment:
I like your pattern of placing utils for particular types in functions named for that type. But as the number grows, should we be relocating these to a `.util` package in the module?

Should we refactor the other utils in the library (in a `util.kt` file iirc) similarly?

Reply:
I don't mind either way.

My pattern is usually
- <Class name>.kt for extension functions for <Class name>
- SomeNameUtils.kt for a collection of utility functions